doc/md/fundamentals/2-types/6-objects-classes.md

```diff
@@ -132,7 +131,7 @@ A function expecting `{ getBalance : () -> Nat }` can accept any of the above, s
 - A subtype must be usable wherever its supertype is expected.
 - The more general object has fewer methods because it makes fewer assumptions about available functionality.
```

Comment:
```suggestion
- In general, if type `T` is a subtype of type `U`, then any value of type `T` also has type `U`. The means that a value of the more specific type `T` can pass as a value of the more general type `U` without any explicit conversion.
- When `T` and `U` are object types, subtyping ensures that `T` provides at least the fields required by `U`, so that any object of type `T` can serve as an object of the more general type `U`. 

The more general object type has fewer fields because this places fewer requirements on its values.
```

This is hard to explain. I hope that improves it a bit.